sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_internal/__init__.py

```diff
@@ -20,7 +20,7 @@
     "KeyVaultClientBase",
 ]
 
-_VaultId = namedtuple("VaultId", ["vault_url", "collection", "name", "version"])
+_VaultId = namedtuple("VaultId", ["vault_url", "collection", "name", "version"])  # type: ignore
```

Comment:
Or just fix the error 😜 
```suggestion
_VaultId = namedtuple("_VaultId", ["vault_url", "collection", "name", "version"])
```